usaspending_api/accounts/views/federal_accounts_v2.py

```diff
@@ -451,12 +458,87 @@ def get(self, request, fed_acct_code, format=None):
                 "parent_agency_toptier_code",
                 "parent_agency_name",
             )
+            .first()
         )
-
         if not federal_account:
-            raise InvalidParameterException("Federal Account with code {} does not exist.".format(fed_acct_code))
+            raise InvalidParameterException(
+                f"Cannot find Federal Account with the code of '{federal_account_code_param}'"
+            )
+        return federal_account
+
+    def _get_treasury_accounts(self) -> List[dict]:
+        """Get the Treasury Accounts (using File A and B) associated with the given Federal Account during a specific
+            fiscal year.
+
+        Returns:
+            List[dict]: List of Treasury Accounts
+        """
+
+        submission_ids = get_latest_submission_ids_for_fiscal_year(self.fiscal_year)
+        query_filters = [
+            Q(submission__reporting_fiscal_year=self.fiscal_year),
```

Comment:
Is this filter on the submission's fiscal year redundant? Same for the submission fiscal year filter in `tbr_subquery`. We're already filtering down to a list of submissions from the provided fiscal year

Reply:
I removed the fiscal_year filter in commit 97dff66fd3b83b3060242d1da14a28312e69c34d since it did appear redundant.